### PR TITLE
Add alternative Spear recipe for Ash walkers

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Add alternative Spear recipe in Primal category with sinew restraints instead of cable restraints

## Why It's Good For The Game
Spear are almost useless in hunting and throwing compare to bone spear, but common spears are ideal tool for funny head lamps. So i added alternatire recipe for this purpose.

## Images of changes
image

## Testing
add new recipe with 1 change. Best coder based PR.

## Changelog
:cl:
add: Add alternative Spear recipe with sinew restraints instead cable coils in Primal section
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
i'd rather see this be merged with the normal spear, having three different spear crafting methods(not counting manual crafting) is a bit much

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -477,6 +477,15 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/spear_primal // alternative recipe for Ash Walkers
+	name = "Common Spear"
```
Comment (DGamerL) [inline code/modules/crafting/recipes.dm:481]:
```suggestion
	name = "Spear"
```
This might be better due to the `Bone spear` already being called that

**Comment by DGamerL** [comment]
> i'd rather see this be merged with the normal spear, having three different spear crafting methods(not counting manual crafting) is a bit much

TLDR; we can't. This is as far as I know the best way to handle multiple ways to make items.
We don't want to include all subtypes of restraints probably, as that includes a fair amount

**Comment by spacegeese** [comment]
> We don't want to include all subtypes of restraints probably, as that includes a fair amount

i'm no coder, but would making sinew a subtype of cables work?

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -477,6 +477,15 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/spear_primal // alternative recipe for Ash Walkers
+	name = "Spear"
+	result = list(/obj/item/spear)
+	reqs = list(/obj/item/restraints/handcuffs/sinew = 1,
+				/obj/item/shard = 1,
+				/obj/item/stack/rods = 1)
+	time = 40
```
Comment (Henri215) [inline code/modules/crafting/recipes.dm:486]:
```suggestion
	time = 4 SECONDS
```

**Review by Henri215 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]
